limiter.go

```diff
@@ -60,3 +62,12 @@ func (limit *simpleLimiter) Count(n int) error {
 	}
 	return nil
 }
+
+// UntilNext returns a channel that can be used to block until next window
+func (limit *simpleLimiter) UntilNext() <-chan time.Time {
```

Comment:
Delay() is maybe a better name but I don't hate this one either. :)

Reply:
changed to delay. :)